### DISCUSSION TITLE
White label Coolify

### DISCUSSION THREAD

**Discussion by VaarunSinha** [coollabsio/coolify#1744] [Feature Requests]
I would like to suggest and contribute to make a feature where we can change the overall color scheme and add the company's logo in Coolify.

**Comment by dmnhld** [comment]
I would also be very happy about such a feature.
I think that business customers would also be much more open to Coolify through white labeling.

**Comment by kikehz** [comment]
I strongly agree, i don't think is something very difficult or time-consuming and is a feature very useful in order to deploy in customers infrastructure

**Comment by oudayfatteh0** [comment]
I'm a little late for this conversation but I think it's intentional to not having white label in coolify. 
If you want I created a modified version of coolify with supported branding customizations + email templates.
It still in development for now but you can try it out, read the docs to install it
https://github.com/oudayfatteh0/coolify-white-label
